app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def order_link(heading, content_item_attribute, order)
+    link_to heading, organisation_content_items_path(
```

Comment:
This doesn't need to be addressed in this PR, but just a heads up of what changes we will need to make to the markup:

We will need to add a class and a title (or otherwise hidden copy), some of it potentially on the th and some on the link. (We might not need the class at all as we could use `[aria-sort] {}` for styling?)
This is some example markup (subject to change, would like to discuss with @theleebriggs):

```
<th class="sort none" aria-sort="none"><a href="..." title="Sort column ascending">Last Updated</a></th>
```
would be the default, and...

```
<th class="sort ascending" aria-sort="asc"><a href="..." title="Sort column descending">Last Updated</a></th>
```
...would be when it's sorted ascending. Sorted descending would be like the latter but the opposite.

I wonder if that means that the `th` should ideally be part of `order_link()`?